File: QE-Python_selenium/qe-selenium-python/Activity16.py
from selenium import webdriver
from selenium.webdriver.support.select import Select

from selenium.webdriver.common.by import By
driver = webdriver.Firefox()
driver.get("https://training-support.net/webelements/selects")


print(driver.title)
dropdown_ele=driver.find_element(By.TAG_NAME,'select') 
dropdown_obj=Select(dropdown_ele)
for opt in dropdown_obj.options:
    print(opt.text)
options=dropdown_obj.options
dropdown_obj.select_by_index(2)
dropdown_obj.select_by_value("four")
dropdown_obj.select_by_visible_text("Three")
driver.quit()

File: QE-Java-Selenium/QE-selenium-java/javaselenium/src/main/java/javaselenium/demos/Activity16.java
package javaselenium.demos;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.openqa.selenium.support.ui.Select;

import java.util.List;

public class Activity16 {
    public static void main(String[] args) {

        WebDriver driver = new FirefoxDriver();
        driver.get("https://training-support.net/webelements/selects"); 
        System.out.println("Page Title: " + driver.getTitle());

        WebElement dropdownEle = driver.findElement(By.tagName("select"));

        Select dropdown = new Select(dropdownEle); 
        List<WebElement> options = dropdown.getOptions();
        System.out.println("\nDropdown options:");
        for (WebElement opt : options) {
            System.out.println(opt.getText());
        }
        dropdown.selectByIndex(2);
        dropdown.selectByValue("four");
        dropdown.selectByVisibleText("Three");

        driver.quit();
    }
}